src/utils/index.js

```diff
@@ -57,3 +57,49 @@ export function slug() {
 export function promiseWait(ms) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
+
+export function wrapContract(contract, name, abi, txManager) {
```

Comment:
i would put this somewhere specific to the smart contract service (e.g. `src/eth/smartContract/wrapContract.js`) instead of in `utils/index`... utils index files have a way of becoming a big mess, like that kitchen drawer you put receipts and hand tools and rubber bands in